src/audio_backend.cpp

```diff
@@ -115,3 +122,49 @@ void RealTimeAudioThread::AdvanceFrames(FramePointers& Frame)
     }
     LastFrameStart = FrameStart;
 }
+
+
+// This no-op stub is used if no AudioStream is available.
+// It's quite possible the program should just crash, instead. What use is an audio generator with no audio?
```

Comment:
A null audio backend is super useful for porting to new platforms, as evident by you adding it :)

Reply:
Also theoretically useful if I ever add the ability to record to files.